cmd/bisync/bisync_test.go

```diff
@@ -157,7 +157,7 @@ var logHoppers = []string{
 var logLinesWithSlash = []string{
 	`.*\(\d\d\)  :.*(fix-names|touch-glob|touch-copy|copy-file|copy-as|copy-dir|delete-file) `,
 	`INFO  : - .*Path[12].* +.*Queue copy to.* Path[12].*`,
-	`INFO  : Synching Path1 .*? with Path2 `,
+	`INFO  : Syncing Path1 .*? with Path2 `,
```

Comment:
Hmm, I think this makes the test fail, as log still reports:

25/01/15 16:09:26 INFO  : Synching Path1 "C:\Users\RUNNER~1\AppData\Local\Temp\160916ci\volatile\path1\" with Path2 "C:\Users\RUNNER~1\AppData\Local\Temp\160916ci\volatile\path2\"

Reply:
Caused by this: https://github.com/DimitriPapadopoulos/rclone/blob/f1ef068f139e3f6015031ca9339c5d577ecfd7a7/cmd/bisync/operations.go#L227

And then there are a lot of of log statements in log files under `cmd/bisync/testdata`, maybe those must/should be updated correspondingly.

Also found one additional occurrence of sync**h** left, in a comment: https://github.com/DimitriPapadopoulos/rclone/blob/f1ef068f139e3f6015031ca9339c5d577ecfd7a7/cmd/bisync/listing.go#L397

Reply:
> And then there are a lot of of log statements in log files under cmd/bisync/testdata, maybe those must/should be updated correspondingly.

Yes they do. I don't like this design but we (@ncw &  @nielash ) haven't come up with anything better yet!

Reply:
Fixed. I don't want to change log messages here, I'd leave that for a major release.